downstream/modules/platform/proc-operator-access-aap.adoc

```diff
@@ -1,3 +1,31 @@
 [id="operator-access-aap_{context}"]
 
-= Accessing your AnsibleAutomationPlatform  
\ No newline at end of file
+= Accessing your central configuration  
+You should use the *AnsibleAutomationPlatform* instance as your default. 
+This instance links the {ControllerName}, {HubName}, and {EDAName} deployments to a single interface. 
+
+.Procedure
+
+To access your *AnsibleAutomationPlatform* instance:
+
+. Navigate to menu:Networking[Routes]
```

Comment:
Flagged in Vale for "Navigate" - Depending on the context, consider using 'click', 'select', 'browse', or 'go to' rather than 'Navigate'